cluster-autoscaler/cloudprovider/mcm/mcm_cloud_provider.go

```diff
@@ -319,6 +319,10 @@ func (machinedeployment *MachineDeployment) DecreaseTargetSize(delta int) error
 	if err != nil {
 		return err
 	}
+	if int(size+int64(delta)) < machinedeployment.minSize {
```

Comment:
instead of doing double conversions you could have done the following:
```suggestion
	if (int)size+ delta < machinedeployment.minSize {
```

Reply:
done